firmware/application/apps/ui_recon.hpp

```diff
@@ -206,13 +127,28 @@ namespace ui {
 			bool load_hamradios = { true };
 			bool update_ranges = { true };
 			bool fwd = { true };
+            bool recon = true ;
 			uint32_t recon_lock_nb_match = { 3 };
-			uint32_t recon_lock_duration = { 50 };
-			uint32_t recon_match_mode = { 0 };
+			uint32_t recon_lock_duration = { RECON_DEF_LOCK_DURATION };
+			uint32_t recon_match_mode = { RECON_MATCH_CONTINUOUS };
 			bool scanner_mode { false };
```

Comment:
It could be an enum, I prefer them as a define.
If you were talking about recon_lock_duration & recon_match_mode they can change so an enum does no fit